released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Numerical Analysis in Coq"
+name: "coq-num-analysis"
+version: "1.0.0"
+maintainer: "MILC project <milc@inria.fr>"
+authors: [ "Sylvie Boldo" "François Clément" "Vincent Martin" "Micaela Mayero" "Florian Faissole" "Houda Mouhcine" "Louise Leclerc" "Stéphane Aubry" ]
+homepage: "https://lipn.univ-paris13.fr/coq-num-analysis/"
+dev-repo: "git+https://lipn.univ-paris13.fr/coq-num-analysis/"
+bug-reports: "https://lipn.univ-paris13.fr/coq-num-analysis/issues"
+license: "LGPL-3.0-or-later"
+tags: [ "keyword:real analysis" "keyword:Lax Milgram" "keyword:Lebesgue" "keyword:Tonelli" "keyword:Bochner"]
```

Comment:
I suggest adding a few more tags for the purpose of accurate metadatas:
- `"category:..."` for proper indexing in  https://coq.inria.fr/opam/www/
- `"date:2022-08-..."` for the release date of the library (not of the Opam package)
- `"logpath:..."` for the Coq logical path where the library gets installed